tests/test-pkcs11-tool-sym-crypt-test.sh

```diff
@@ -181,6 +181,26 @@ assert $? "Fail/pkcs11-tool decrypt"
 cmp gcm_vector_plain.data gcm_test_plain.data >/dev/null 2>&1
 assert $? "Fail, AES-GCM - wrong decrypt"
 
+echo "======================================================="
+echo " AES-CTR"
+echo " OpenSSL encrypt, pkcs11-tool decrypt"
+echo " pkcs11-tool encrypt, compare to openssl encrypt"
+echo "======================================================="
+
+openssl enc -aes-128-CTR -nopad -in aes_plain.data -out aes_ciphertext_openssl.data -iv "${VECTOR}" -K "70707070707070707070707070707070"
```

Comment:
sounds like libressl needs the ctr in lower case from the failure in the build-libressl job?
https://github.com/OpenSC/OpenSC/actions/runs/13284043279/job/37088623747?pr=3338